Makefile

```diff
@@ -76,32 +84,41 @@ dep-update:
 
 # cloud targets - building images and deploying
 
-docker:
+.get-console:
 	@mkdir -p _output/docker
-	@cp -r deploy/docker/* _output/docker
-	@cp ${GOPATH}/bin/sws _output/docker
+ifeq ("${CONSOLE_VERSION}", "local")
+	echo "Copying local console files from ${CONSOLE_LOCAL_DIR}"
+	rm -rf _output/docker/console && mkdir _output/docker/console
+	cp -r ${CONSOLE_LOCAL_DIR}/build/* _output/docker/console
+else
 	@if [ ! -d "_output/docker/console" ]; then \
```

Comment:
I think this is to cache the previous download, but we will need something to know if `_output/docker/console` holds the value of specified ${CONSOLE_VERSION} or some old value (local code, previous version, etc).

```bash
# Make docker with local
CONSOLE_VERSION=local make docker
# Try to make docker with version=0.2.3  In reality we are using our old _output/docker/console dir
CONSOLE_VERSION=0.2.3 make docker
```

Reply:
yeah, that is a shortcoming. You have to make clean if you want to switch versions. I didn't bother - I assume the user will know when he is switching between versions and will know to clean. For now, I thought this was good enough. I didn't want to have to code up everything to figure out automatically if the versions matched

Reply:
I ok with that, but warning on the comments above `CONSOLE_VERSION ?= latest` would be great.

Reply:
comment added - see the commit #7565c41